shard-operator-integration-tests/src/test/java/com/redhat/service/bridge/shard/operator/cucumber/steps/BridgeIngressSteps.java

```diff
@@ -26,9 +29,22 @@ public BridgeIngressSteps(Context context) {
     }
 
     @When("^deploy BridgeIngress:$")
-    public void deployBridgeIngress(String bridgeIngressYaml) {
+    public BridgeIngress deployBridgeIngress(String bridgeIngressYaml) {
```

Comment:
If this step is replaced by the following one then you can merge them, no need to keep here a step which is not used.